app/Http/Controllers/OrderManagementController.php

```diff
@@ -0,0 +1,157 @@
+        }
+
+        // 注文した料理のキャンセル
+        try {
+            OrderedDishManagementService::cancelOrderedDish($orderedDishId);
+        } catch (\Exception $e) {
+            return response()->json(['error' => $e->getMessage()], 500);
+        }
+
+        return response()->json(['message' => '注文した料理をキャンセルしました'], 200);
+    }
+
+    /**
+     * 注文ステータスを提供済みにする
+     * @param string $orderedDishId 注文した料理の注文ID (required)
+     * @return \Illuminate\Http\JSONResponse
+     */
+    public function orderedDishDelivery(string $orderedDishId)
+    {
+        // 注文した料理が存在するか確認(存在しない場合は、400エラーを返す)
+        if (!OrderedDishManagementService::isExist($orderedDishId)) {
+            return response()->json(['error' => '指定した料理は存在しません'], 400);
+        }
+
+        // 注文した料理のステータスを提供済みにする
+        try {
+            OrderedDishManagementService::deliverOrderedDish($orderedDishId);
+        } catch (\Exception $e) {
+            return response()->json(['error' => '注文した料理のステータスの更新に失敗しました'], 500);
+        }
+
+        return response()->json(['message' => '注文した料理のステータスを提供済みにしました'], 200);
+    }
+
+    /**
+     * 指定した店舗の注文一覧の取得
+     * @param string $restaurantId 注文一覧を取得したい店舗のID (required)
+     * @return \Illuminate\Http\JSONResponse
+     */
+    public function orderList(string $restaurantId)
+    {
+        // 指定した店舗が存在するか確認(存在しない場合は、400エラーを返す)
+        if (!RestaurantManagementService::isExist($restaurantId)) {
+            return response()->json(['error' => '指定した店舗は存在しません'], 400);
+        }
+
+        // 注文一覧の取得
+        $orderList = OrderManagementService::getOrderList($restaurantId);
+
+        return response()->json($orderList, 200);
+    }
+
+    /**
+     * 指定した店舗の未提供料理一覧を取得 (注文時間順)
+     * @param string $restaurantId 未提供料理一覧を取得したい店舗のID (required)
+     * @return \Illuminate\Http\JSONResponse
+     */
+    public function unservedDishList(string $restaurantId)
+    {
+        // 指定した店舗が存在するか確認(存在しない場合は、400エラーを返す)
+        if (!RestaurantManagementService::isExist($restaurantId)) {
+            return response()->json(['error' => '指定した店舗は存在しません'], 400);
+        }
+
+        // 未提供料理一覧の取得
+        try {
+            $unservedDishList = OrderedDishManagementService::getUnservedDishListByRestaurant($restaurantId);
+        } catch (\Exception $e) {
+            return response()->json(['error' => $e->getMessage()], 500);
+        }
+
+        return response()->json($unservedDishList, 200);
+    }
+
+    /**
+     * 注文詳細の取得
+     * @param string $orderId 注文詳細を取得したい注文のID (required)
+     * @return \Illuminate\Http\JSONResponse
+     */
+    public function orderGet(string $orderId)
+    {
+        // 指定した注文が存在するか確認(存在しない場合は、400エラーを返す)
+        if (!OrderManagementService::isExist($orderId)) {
+            return response()->json(['error' => '指定した注文は存在しません'], 400);
+        }
+
+        // 注文詳細の取得
+        $order = OrderManagementService::getOrderInfo($orderId);
+
+        return response()->json($order, 200);
```

Comment:
```suggestion
        try {
            // 注文詳細の取得
            $order = OrderManagementService::getOrderInfo($orderId);
        } catch (OrderNotFoundException $e) {
            return response()->json(['error' => '指定した注文は存在しません'], 400);
        }
        return response()->json($order, 200);
```

[IMO] **アプリケーション設計**
DBの存在確認 → データの取得、とすると、DBへ2回アクセスが走ってしまいます。
パフォーマンスのことを考えるなら、このような設計にしておくと良いかなと思います。（`OrderNotFoundException`は勝手に作ったクラスです）